WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -396,18 +361,15 @@ class MySiteViewModelTest : BaseUnitTest() {
     fun `when no site is selected and screen height is higher than 600 pixels, show empty view image`() {
         whenever(displayUtilsWrapper.getDisplayPixelHeight()).thenReturn(600)
 
-        initSelectedSite()
         onSiteSelected.value = null
 
         assertThat(uiModels.last().state).isInstanceOf(NoSites::class.java)
         assertThat((uiModels.last().state as NoSites).shouldShowImage).isTrue
     }
 
-    @Test
-    fun `when no site is selected and screen height is lower than 600 pixels, hide empty view image`() {
+    @Test fun `when no site is selected and screen height is lower than 600 pixels, hide empty view image`() {
```

Comment:
Minor (🔍): Consider moving the `@Test` annotation to its own line.

Reply:
Oops, one slipped through - I had gone through each tests one-by-one to fix all the broken ones. Thanks!